### DISCUSSION TITLE
integrating/merging two citeseq samples

### DISCUSSION THREAD

**Discussion by jessielyons** [satijalab/seurat#4814] [Q&A] [answered]
Hi Seurat,

I have two 10x generated samples where I used cell ranger multi command to run gex, totalseqc (adt) and tcr_t. I am using seurat for downstream analysis. 

Can you guide me how I can merge/integrate the two samples for both gex and adt  in Seurat? The vignette does not help at all with multi sample integration of gex and adt datasets. It only talks about gex integration alone.

samples <- c("Sample1testmatrix","Sample2testmatrix")

for (input in samples){
  seurat_obj <- Read10X(data.dir = paste0("home/OUTPUTS/", input))
  seurat_data <- CreateSeuratObject(counts = seurat_obj$`Gene Expression`,
                                   project = file)
  seurat_data[['Protein']] = CreateAssayObject(counts = seurat_obj$`Antibody Capture`)
  assign(input, seurat_data)
}
 
Thanks,
Jessie

**Comment by timoast** [comment, marked as answer]
One way to do this is to find anchors between your samples using one modality (RNA for example), and use those anchors to integrate both the RNA and protein modalities. For example, you can run `IntegrateData()` to integrate the RNA assay, and run `IntegrateEmbeddings()` to integrate the PCA derived from the protein measurements from each sample, and then add the corrected dimension reduction to the integrated object.

**Reply by coopershawna** [reply]
Thank you!  
How does that then work for doing differential expression since we've introduced dependencies? Are we just using the similar trick from multiome for comparisons between case and control within clusters (1_WT vs 1_KO, ect)?

**Reply by timoast** [reply]
For differential expression / differential accessibility you always want to use the original (uncorrected) data. If you have multiple replicates per condition, you can include replicate as a latent variable in the model.

**Reply by coopershawna** [reply]
Ok. Perfect! Thank you!

**Comment by jessielyons** [comment]
Hi @timoast and @yuhanH , Now I have a similar issue with how to integrate multiple samples while using HTO enrichment vignette? It's easy to integrate RNA part. Can you please provide guidelines on how to integrate HTO data coming from multiple samples? When do we run HTODemux? Do we do HTODemux individually on all samples before we run "IntegrateData" on HTO data? Please guide and thanks a ton!
